extensions/Worker.Extensions.Http.AspNetCore/src/ExtensionStartup.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Extensions.Http.AspNetCore;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Azure.Functions.Worker.Core.Http;
+using Microsoft.Azure.Functions.Worker.Pipeline;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+[assembly: WorkerExtensionStartup(typeof(ExtensionStartup))]
+
+namespace Microsoft.Azure.Functions.Extensions.Http.AspNetCore
+{
+    internal class ExtensionStartup : WorkerExtensionStartup
```

Comment:
I think this needs to be public for the generator - getting an error due to the protection level.

Reply:
Yes. Opened #1404 to track enhancements to the experience, but taking a different approach with the changes.